test/NPacMan.Game.Tests/GameTests/GameTests.cs

```diff
@@ -0,0 +1,42 @@
+using FluentAssertions;
+using Xunit;
+
+namespace NPacMan.Game.Tests.GameTests
+{
+    public class GameTestsxx
+    {
+        private readonly TestGameSettings _gameSettings;
+        private readonly Game _game;
+
+        public GameTestsxx()
```

Comment:
Should this be GameTests?

Reply:
Yeah, I renamed the file while moving everything around as it conflicted with the namespace. I'll rename it back 👍